services/core/entities/demoReviewS3LocationEntity.ts

```diff
@@ -0,0 +1,21 @@
+import { Entity } from "dynamodb-toolbox";
+import { getTable } from "./getTable";
+
+export const getDemoReviewS3LocationEntity = (tableName: string): Entity => {
+  const myTable = getTable(tableName);
+
+  const demoReviewS3LocationEntity = new Entity({
+    name: "demoReviewS3LocationEntity",
+    attributes: {
+      PK: { partitionKey: true, hidden: true, prefix: "DEMOREVIEWID#" },
+      SK: { sortKey: true, hidden: true, default: "S3_LOCATION" },
+      reviewId: ["PK", 0, { type: "string", required: true }],
+      inputLocation: { type: "string", required: true },
+      responseLocation: { type: "string", required: true },
+      date: { type: "string", default: () => new Date().toDateString() },
+    },
+    table: myTable,
+  } as const);
```

Comment:
I think this entity should be called ReviewEntity. It can be the same for demo and not. it should have an attribute which is a boolean which differentiates.